src/bun.js/bindings/BunProcess.cpp

```diff
@@ -1916,6 +1933,29 @@ static JSValue constructProcessSend(VM& vm, JSObject* processObject)
     }
 }
 
+JSC_DEFINE_HOST_FUNCTION(processDisonnectFinish, (JSGlobalObject * globalObject, JSC::CallFrame* callFrame))
+{
+    Bun__closeChildIPC(globalObject);
+    return {};
```

Comment:
This function will always crash. I don't see a test failure

Can you add a test?

Reply:
in what condition do you see this as being able to cause a crash?

Reply:
clarified offline, `return {}` is problematic if an exception hasnt been thrown